### PR TITLE
Updated to Vapor 2

### PR DESCRIPTION
**Pull request by valen90**

### CODE REVIEW

File: Sources/Storage/Provider.swift [hunk 1]
```diff
@@ -29,6 +28,8 @@ public final class StorageProvider: Provider {
     }
     
     public func boot(_ drop: Droplet) {}
```
Comment (steffendsommer) [inline Sources/Storage/Provider.swift]: Is this still required or could it be removed?

Comment (valen90) [inline Sources/Storage/Provider.swift]: Is still required in order to Conform to Provider

**Comment by a2** [comment]
Vapor 2.0 was released in the last couple of days. Is there anything I can do to help prepare Storage for an update?

**Comment by steffendsommer** [comment]
@a2 we will get this in now, then I'll remove the pre-release identifiers and get it released. Thanks though 👍

**Pull request merged**